lms/views/dashboard/api/assignment.py

```diff
@@ -50,10 +52,9 @@ class AssignmentsMetricsSchema(PyramidRequestSchema):
 class AssignmentViews:
     def __init__(self, request) -> None:
         self.request = request
-        self.h_api = request.find_service(HAPI)
+        self.h_api: HAPI = request.find_service(HAPI)
```

Comment:
Include the type anno so mypy is aware of the return values of the methods of this service.